Content.Server/Abilities/Boxer/BoxingSystem.cs

```diff
@@ -0,0 +1,111 @@
+                return;
+            }
+            Box(args.HitEntities, modifier: boxModifier);
+        }
+
+        private void ApplyBoxerModifiers(EntityUid uid, BoxerComponent component, MeleeHitEvent args)
+        {
+            args.ModifiersList.Add(component.UnarmedModifiers);
+        }
+
+        private void OnEquipped(EntityUid uid, BoxingGlovesComponent component, GotEquippedEvent args)
+        {
+            // This only works on clothing
+            if (!TryComp<ClothingComponent>(uid, out var clothing))
+                return;
+            // Is the clothing in its actual slot?
+            if (!clothing.SlotFlags.HasFlag(args.SlotFlags))
+                return;
+
+            // Set the component to active to the unequip check isn't CBT
+            component.IsActive = true;
+
+            EnsureComp<BoxingComponent>(args.Equipee);
+        }
+
+        private void OnUnequipped(EntityUid uid, BoxingGlovesComponent component, GotUnequippedEvent args)
+        {
+            // Only undo the resistance if it was affecting the user
+            if (!component.IsActive)
+                return;
+            component.IsActive = false;
+            RemComp<BoxingComponent>(args.Equipee);
+        }
+
+
+        private void Box(IEnumerable<EntityUid> hitEntities, float paralyzeTime = 5f, float paralyzeChanceNoSlowdown = 0.2f,
+            float slowdownTime = 3f, float paralyzeChanceWithSlowdown = 0.5f, float modifier = 1f)
+        {
+            foreach (var entity in hitEntities)
+            {
+                if (!TryComp<StatusEffectsComponent>(entity, out var status))
+                    continue;
+
+                if (HasComp<KnockedDownComponent>(entity))
+                    return;
+
+                if (!HasComp<SlowedDownComponent>(entity))
+                {
+                    if (_robustRandom.Prob(paralyzeChanceNoSlowdown * modifier))
+                    {
+                        SoundSystem.Play("/Audio/Weapons/boxingbell.ogg", Filter.Pvs(entity), entity);
+                        _stunSystem.TryParalyze(entity, TimeSpan.FromSeconds(paralyzeTime * modifier), true, status);
+                    }
+                    else
+                        _stunSystem.TrySlowdown(entity, TimeSpan.FromSeconds(slowdownTime * modifier), true,  0.5f, 0.5f, status);
+                }
+                else
+                {
+                    if (_robustRandom.Prob(paralyzeChanceWithSlowdown * modifier))
+                    {
+                        SoundSystem.Play("/Audio/Weapons/boxingbell.ogg", Filter.Pvs(entity),  entity);
+                        _stunSystem.TryParalyze(entity, TimeSpan.FromSeconds(paralyzeTime * modifier), true, status);
+                    }
+                    else
+                        _stunSystem.TrySlowdown(entity, TimeSpan.FromSeconds(slowdownTime * modifier), true,  0.5f, 0.5f, status);
+                }
+            }
```

Comment:
As above.

Reply:
I guess we should wait for stamina system? Not sure how else you'd do it